### ISSUE TITLE
No option to save data to the ROM

### ISSUE THREAD

**Issue by vince94** [Kermalis/GBAMusicStudio#20]
Hello! This is my first time using the program, so apologies if it's obvious and I'm just not seeing it. I'm exporting to MIDI, changing the arrangement in Sekaiju, and opening the MIDI back up to preview it, but I can't find an option to save modified + converted MIDI data back to the ROM. Is this functionality something that's currently being worked on/isn't complete yet?

Also, this is a tangent, but I saw in the README.md that you might support Amiga MODs in the future if you go insane. If I had the know-how, I'd use .XM files or Impuse Tracker .IT files instead - there are a lot less weird limitations in those file formats than the Amiga MOD format (forced panning, four-channel limit, etc.)

**Comment by Kermalis** [comment]
Sorry for the late response, there is no saving back to ROM yet. Either way with the MOD format, if I support it, importing MOD tracks would mean emulating the music in them as the M4A engine, so the forced panning would be brought with it and there wouldn't be issues. I'll look into the XM and IT formats as well, but I've never heard of them before.

**Comment by vince94** [comment]
Yeah, I was just saying that the forced panning, channel limit, and such is way too restrictive to be worth using imo, at least as far as the GBA is concerned. If you decide to look into it, you can find more information about the formats [here](ftp://ftp.modland.com/pub/documents/format_documentation/FastTracker%202%20v2.04%20(.xm).html) and [here](https://github.com/schismtracker/schismtracker/wiki/ITTECH.TXT), and you can find example files to experiment with on the [OpenMPT forum](https://forum.openmpt.org/index.php?board=6.0).

**Comment by Kermalis** [comment]
Thanks for the links
